### DISCUSSION TITLE
Datatracker home page looks like a parking page on mobile browsers

### DISCUSSION THREAD

**Discussion by kesara** [ietf-tools/datatracker#4282] [General]
I think the datatracker mobile front page should put more visible links. To me, this comes out as a parking page. Because the hamburger button is mixed with the header colours. And the only viable link that comes out is the [www.ietf.org](http://www.ietf.org/)
Also, the current page doesn't indicate that there is content below.

Screenshot from iOS Safari on iPhone 13 mini:
![IMG_0218](https://user-images.githubusercontent.com/1027692/181074753-5e737abf-5301-46e2-b1bb-a8fbe501870e.PNG)

**Comment by rjsparks** [comment]
cc @larseggert @ghwood

**Comment by ghwood** [comment]
To a different extent, this is an issue even with a viewport of a pretty good size (~1200px x 600px screenshot attached). 

Rethinking the front page concept entirely (not just the mobile edition) would be worthwhile, IMO.

<img width="1252" alt="image" src="https://user-images.githubusercontent.com/5347540/181076731-2e49c6d3-c592-44a5-86a8-98674506f1d7.png">

**Comment by NGPixel** [comment]
How about something like a dashboard with overview cards from each section?

<img width="991" alt="Screenshot 2022-07-27 022845" src="https://user-images.githubusercontent.com/15522395/181177074-24dbb9f4-6f83-4b0c-a467-db9bc9ef6178.png">
